M2/Macaulay2/e/f4/ntuple-monomial.hpp

```diff
@@ -15,51 +15,6 @@ typedef ntuple_monomials::Exponent ntuple_word;
 typedef ntuple_word *ntuple_monomial;
 typedef const ntuple_word *const_ntuple_monomial;
 
-template <>
```

Comment:
I take it the code in this file was removed as it was not used, or is no longer used?

Reply:
I'm not sure if it was used (I suspect not), but they are all still covered by the template specialization, so you can still use them just as before.